src/python/WMCore/WMSpec/Steps/Template.py

```diff
@@ -62,6 +63,22 @@ def environment(self):
         """
         return self.data.environment
 
+    def setOverrides(self, **overrides):
```

Comment:
@germanfgv  I think (similarly as bellow) an equivalent of this method is already present into the object:
 https://github.com/dmwm/WMCore/blob/0d667606fed0166f304f48c2fc85b9c3b32cc0f9/src/python/WMCore/WMSpec/WMStep.py#L157

Of course the already existing one is having the unpleasant need to call it for setting every single override parameter one by one. But if you insist on having the option to pass those `overrides` on a single pas as a dictionary (which I personally find quite useful), please  move it to the proper place. i.e. here:

https://github.com/dmwm/WMCore/blob/0d667606fed0166f304f48c2fc85b9c3b32cc0f9/src/python/WMCore/WMSpec/WMStep.py#L168

Reply:
You are right, I missed this.